Templates/project_collection/README.md

```diff
@@ -0,0 +1,212 @@
+# Instructions for filling out Project Collection Template
+The project and data collection template contains all the information necessary
+to add a new project and/or data collection to our inventory. There are three
+sheets in this Excel document: `Data collection project`, `Data collection`,
+and `values`. The first two sheets are to be filled out with the appropriate
+information about your project and data collection. The third sheet contains
+controlled vocabulary and should not be modified.
+
+If you have any questions, please feel free to email
+[data.curation@alleninstitute.org](mailto:data.curation@alleninstitute.org)
+so we can assist you with adding your metadata to the project inventory.
+
+
+Please refer to the following descriptions of values when filling `project_collection_template.xlsx`:
+
+## Data collection project
+This sheet contains information about your data collection project
+
+#### **Project** - *project name* 
+  * This is a unique human-readable identifier (i.e. handle) used to keep track 
+    of your project
+  * By convention, we prefer to use names in `snake_case_with_proj` as a suffix
+  * If you are not sure what project name to use, please email 
```

Comment:
Do we want to either say more strongly that we will assign the project name, or do we want them to be able to self-select and communicate that we will review and potentially edit this.

Reply:
I'm not sure. Having us assign a name is more work for us, but potentially less confusing.

Reply:
We can say they can suggest a name and we reserve the right to change

Reply:
Edited to reflect the process for reserving project and collection names.